src/OpenTelemetry.Instrumentation.ConfluentKafka/OpenTelemetryConsumeResultExtensions.cs

```diff
@@ -0,0 +1,161 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+using System.Diagnostics;
+using System.Text;
+using Confluent.Kafka;
+using OpenTelemetry.Context.Propagation;
+using OpenTelemetry.Trace;
+
+namespace OpenTelemetry.Instrumentation.ConfluentKafka;
```

Comment:
Should this go in the namespace for `ConsumeResult<TKey, TValue>`? That would make it more discoverable. But we may not want it to be easily discoverable 😄

Reply:
Good idea. I moved the class `OpenTelemetryConsumeResultExtensions` to `Confluent.Kafka namespace`.